app/controllers/concerns/trim_params.rb

```diff
@@ -14,16 +14,28 @@ def strip_parameter_whitespace
   def strip_whitespace(obj)
     case obj
     when Hash
-      obj.inject(obj.class.new) { |h, (k, v)|
-        h[k] = strip_whitespace(v)
-        h
-      }
+      hash?(obj)
     when Array
-      obj.map { |v| strip_whitespace(v) }
+      array?(obj)
     when String
-      obj.strip
+      string?(obj)
     else
       obj
     end
   end
+
+  def hash?(obj)
```

Comment:
Feels a little strange to have these look like predicate methods, I'd rename them.